docs/tutorial/represented-file.md

```diff
@@ -1,28 +1,42 @@
 # Represented File for macOS BrowserWindows
 
-On macOS a window can set its represented file, so the file's icon can show in
-the title bar and when users Command-Click or Control-Click on the title a path
-popup will show.
+## Overview
 
-You can also set the edited state of a window so that the file icon can indicate
-whether the document in this window has been modified.
-
-__Represented file popup menu:__
+On macOS, a window can set its represented file, so the file's icon can be
+shown in the title bar. When users click on the title with `Command` or
+`Control` key pressed, a popup with a path to the file will be shown.
 
 ![Represented File][represented-image]
 
+You can also set the edited state for a window so that the file icon can
+indicate whether the document in this window has been modified.
+
 To set the represented file of window, you can use the
 [BrowserWindow.setRepresentedFilename][setrepresentedfilename] and
-[BrowserWindow.setDocumentEdited][setdocumentedited] APIs:
+[BrowserWindow.setDocumentEdited][setdocumentedited] APIs.
+
+## Example
+
+Starting with a working application from the
+[Quick Start Guide](quick-start.md), add the following lines to the
+`main.js` file:
 
 ```javascript
-const { BrowserWindow } = require('electron')
+const { app, BrowserWindow } = require('electron')
 
-const win = new BrowserWindow()
-win.setRepresentedFilename('/etc/passwd')
-win.setDocumentEdited(true)
+app.whenReady().then(() => {
+  const win = new BrowserWindow()
+
+  win.setRepresentedFilename('/etc/passwd')
+  win.setDocumentEdited(true)
+})
 ```
 
+After launching the Electron application, click on the title with `Command` or
+`Control` key pressed. You should see a popup with the file you just defined:
+
+![Represented file](../images/represented-file.png)
```

Comment:
This would be total waste of space as the whole window is blank
![Screen Shot 2020-10-20 at 10 48 44 PM](https://user-images.githubusercontent.com/16765690/96637028-cc2b3200-1326-11eb-8f47-0ed017b83ea7.png)